froide/templates/mfa/create_TOTP.html

```diff
@@ -0,0 +1,35 @@
+{% extends "mfa/base_create.html" %}
+
+{% load static mfa %}
+{% load i18n %}
+{% load form_helper %}
+
+{% block title %}{% translate "Setup authenticator app" %}{% endblock %}
+
+{% block mfa_heading %}{% translate "Setup authenticator app" %}{% endblock %}
+
+
+{% block mfa_inner %}
+
+<ol class="mt-3">
+  <li>
+    {% translate "Scan the code with your authenticator app (e.g. Google Authenticator or Authy)." %}
```

Comment:
```suggestion
    {% translate "Scan the code with your authenticator app (e.g. Google Authenticator, Aegis or Authy)." %}
```

Let's maybe mention at least one FLOSS alternative… :upside_down_face: